### DISCUSSION TITLE
'Cannot execute code, session has been disposed. Please try restarting the Kernel.'

### DISCUSSION THREAD

**Discussion by Bling16** [blue-yonder/tsfresh#1026] [Q&A]
Since the past few days, every time I run tsfresh codes, I am facing an error causing my Kernel to crash. This is the error which is popping up : 

_Cannot execute code, session has been disposed. Please try restarting the Kernel.
The Kernel crashed while executing code in the the current cell or a previous cell. Please review the code in the cell(s) to identify a possible cause of the failure. Click [here](https://aka.ms/vscodeJupyterKernelCrash) for more info. View Jupyter [log](command:jupyter.viewOutput) for further details._

Basic imports from tsfresh has been throwing this error. All code cells before this run well, but once I run the tsfresh codecell, this error pops up and the Kernel crashes.
![Screenshot 2023-06-09 110858](https://github.com/blue-yonder/tsfresh/assets/133252530/a663d7a2-34ee-4e14-9661-e63a63a825f9)

I have also uploaded a screenshot of what the log files show me. 
![Log files output](https://github.com/blue-yonder/tsfresh/assets/133252530/34848388-568b-400c-8608-d4b4863aa6ba)

I have surfed online and seen possible solutions, but none seem to work. I am not using tensorflow/pytorch in the code - just basic feature extraction using tsfresh. 

Has anyone else been facing the same?

**Comment by nils-braun** [comment]
Have you tried setting up a new python environment (depending on what you are using, either new conda environment or new virtualenv or alike)?
